### ISSUE TITLE
this func supposed to do anything?

### ISSUE THREAD

**Issue by tamerNasser** [facn6/Week5-sam-ameer-make-notes#33]
https://github.com/facn6/Week5-sam-ameer-make-notes/blob/4aa415e65c06173d3c006f169cb546aece9a7f4d/public/app.js#L17-L22

**Comment by samjam48** [comment]
yes it gets the transcript files ready for them to be shown to the user...

**Comment by tamerNasser** [comment]
Why are you using 'window.location.href'? you could just use '/'
Why do you have .then if you are not doing anything?

**Comment by samjam48** [comment]
ahh. being silly on the window.location.href... cheers!

The function is the complete requirements to load the transcript into the front end. the .then waits for the transcript. All that is required is for ameer to take the transcript file and add it to the dom. Without the .then there would be no transcript file.

_(this has now been implemented so the file is now shown In the dom)_
